Dockerfile

```diff
@@ -11,6 +11,7 @@ FROM    source as build
 WORKDIR /src/Debug
 RUN     chmod +x /src/scripts/install-powershell-docker.sh
 RUN     /src/scripts/install-powershell-docker.sh
+ENV	    PATH="/root/.dotnet/tools:${PATH}"
```

Comment:
Can you line this up with the rest please?

Reply:
😢 VS Code had made the first character a tab rather then a space... lol